### DISCUSSION TITLE
GeoJsonLayer line pattern

### DISCUSSION THREAD

**Discussion by radoslavirha** [visgl/deck.gl#6622] [Q&A]
Hello,

is it possible, or will be to use some .png, .svg,... pattern for linestrings? Something similar to `pointType:icon`.

Usage: advanced geojson styling, you don't want just 5px red line, but let's say repeating arrows, or dashed lines,...

Thanks

**Comment by ibgreen** [comment]
I believe dashed lines are supported by deck.gl.

Not a direct answer, but nebula.gl has a path marker layer with repeating arrows.

**Reply by radoslavirha** [reply]
Thank you for your answer. Yeah I know about nebula.gl, but it's not working fine right now. I wrote only basic examples , but what about advanced patterns to visualize e.g. electricity, drainage, roads,.... Inspired by some CAD standards. User can define own .png patterns (mapping) and maybe use it like `pointType:icon` ... line.property = `electricity` I'll use `electricity` from mapping

**Comment by ibgreen** [comment]
I believe there is finally some work on restoring nebula.gl ongoing, keep an eye on it. And if you are willing to get your hands a little bit dirty you can easily copy the PathMarkerLayer out of nebula.gl and into your own app. Obviously the arrow geometry it uses can be replaced with other types of geometries.
